examples_utils/benchmarks/platform_assessment/run_paperspace_tests.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2023 Graphcore Ltd. All rights reserved.
+#!/usr/bin/env -S python3 -u
+import argparse
+import datetime
+import os
+
+
+def upload_report(api_key, folder, dataset, version):
+    """Upload reports to gradient"""
+    os.system(f"gradient apiKey ${api_key}")
+    # for file in `find ${5} -name "*.tar.gz"`
+    for file in os.listdir(folder):
+        if "tar.gz" in file:
+            os.system(
+                f"gradient datasets files put \
+                      --id ${dataset}:${version} --source-path ${file}"
+            )
+            os.system(
+                f"gradient datasets versions commit \
+                      --id ${dataset}:${version}"
+            )
+
+
+def run_platform_assessment(folder, version, spec, other=""):
+    """Run platform_assesment tests"""
+    os.system("git submodule update -j 5 --init")
+    cur_time = datetime.datetime.now().strftime("%Y-%m-%d-%H_%M_%S")
+    log_dir = f"{folder}/log_{version}_{cur_time}"
+
+    os.system(
+        f"\
+        export VIRTUAL_ENV=/some/fake/venv/GC-automated-paperspace-test-{version} &&\
+        python -m examples_utils platform_assessment --spec {spec} {other}\
+        --log-dir {log_dir} \
+        --gc-monitor \
+        --cloning-directory /tmp/clones \
+        --additional-metrics \
+    "
```

Comment:
As this is a direct port of a bash script the call to `os.system` is fine, but we could look at removing it and call the internal functions directly.